src/main/java/frc/robot/operatorInputs/OperatorInputs.java

```diff
@@ -27,7 +27,7 @@ public OperatorInputs(Controls driverControls, Controls operatorControls, Drive
 
     // Driver commands
     driverControls.getJoystickBButton()
-        .whenPressed(new IndexerOutakeCommand(indexerController, shooterSpeedController));
+        .whenHeld(new IndexerOutakeCommand(indexerController));
     driverControls.getJoystickAButton().whileHeld(new PowerCellScoringCommandGroup(drive, limelight,
```

Comment:
What do you think of making intake (scoring) and outtake use bumpers instead of buttons?

Reply:
sounds good

Reply:
I made the outtake use the right bumper, but we should ask Eliza what she wants for scoring.  She set it at A a long time ago, and she's the one controlling the robot